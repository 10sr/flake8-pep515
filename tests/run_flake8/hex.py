c = 0xCAFE_F00D
c = 0xCAFEF_00D
