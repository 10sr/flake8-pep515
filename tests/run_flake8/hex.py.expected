./hex.py:2:16: NSP031 HEX Invalid
